src/test/java/com/okta/tools/aws/settings/CredentialsTest.java

```diff
@@ -50,14 +49,6 @@
                     null, 0, null,
                     null, "_custom");
 
-    /*
-     * Test instantiating a Credentials object with invalid INI.
-     */
-    @Test
-    void instantiateInvalidCredentials() {
-        assertThrows(IOException.class, () -> new Credentials(new StringReader("someinvalidini")));
-    }
-
```

Comment:
This is technically a valid INI file. I couldn't figure out how to construct an invalid INI file. Ini4j got this wrong.